src/adoptawidget/index.md

```diff
@@ -0,0 +1,280 @@
+---
+title: Adopt a Widget
+layout: default
+show_banner: False
+---
+
+<img src="/adoptawidget/images/image1.png" alt="Adopt a Widget logo" width="100%" />
+
+Flutter is an open source project hosted [on
+GitHub](https://github.com/flutter/flutter), and making the documentation better
+helps everyone! So, for the month of November, we’re extending a special
+invitation to contribute to the Flutter project by choosing a Widget and
+improving its API documentation.
+
+## How to contribute {#how-to-contribute}
+
+1. Go to [this list of
+issues](https://github.com/flutter/flutter/issues?q=is%3Aopen+is%3Aissue+label%3A%22adopt+a+widget%22+no%3Aassignee)
+and choose an issue. You can only be assigned one Adopt A Widget issue at a
+time, so pick your favorite!
+2. Leave a comment on the issue, asking to be assigned. The first person to
+respond will be assigned. For example:
+
+<img src="/adoptawidget/images/image2.png" alt="Ask to be assigned" width="100%" />
+
+3. Once you are assigned, set up your environment, make the change, and submit
+the pull request. If you're not sure how to do this, see the
+[Quickstart](#quickstart) below.
```

Comment:
Notice how the generated list goes: 1, 2, 1.

This is because you inserted an image on item #2. To get around this, I drop into HTML. So, it would look like:

<ol markdown="1">
<li markdown="1">Go to...
</li>

<li markdown="1">Leave a comment...
</li>

<li markdown="1">Once you are assigned...
</li>

</ol>